### PR TITLE
Update s3 integration with instructions for request level metrics

### PR DESCRIPTION
**Pull request by DanielLanger**

### CODE REVIEW

File: content/integrations/awss3.md [hunk 1]
```diff
@@ -9,15 +9,25 @@ git_integration_title: amazon_s3
...
 Enable this integration to see in Datadog all your S3 metrics.
 
 # Installation
 
-The only requirement for this integration is the permission `s3:GetBucketTagging`.
+* **Daily Storage Metrics**
+
+	The only requirement for this integration is the permission `s3:GetBucketTagging`.
+
+* **Request Metrics**
```
Comment (technovangelist) [inline content/integrations/awss3.md]: im confused. if the only thing needed is the permission, whats the other step for?

Comment (DanielLanger) [inline content/integrations/awss3.md]: I guess I should reword that. If you only want daily S3 metrics, you only have to have that permission in the AWS policy

**Comment by DanielLanger** [comment]
@technovangelist updated

File: content/integrations/awss3.md [hunk 2]
```diff
@@ -9,15 +9,25 @@ git_integration_title: amazon_s3
 
 # Overview
 
+![RDS Dashboard](/static/images/s3_db_screenshot.png)
```
Comment (irabinovitch) [inline content/integrations/awss3.md]: Should this be S3 Dashboard?

**Pull request merged**